prow.go

```diff
@@ -650,6 +649,9 @@ func (m *jobManager) waitForJob(job *Job) error {
 	job.PasswordSnippet = strings.TrimSpace(reFixLines.ReplaceAllString(string(logs), "$1"))
 
 	password, err := commandContents(m.coreClient.CoreV1(), m.coreConfig, namespace, targetPodName, "test", []string{"cat", "/tmp/artifacts/installer/auth/kubeadmin-password"})
+	if err != nil {
+		password, err = commandContents(m.coreClient.CoreV1(), m.coreConfig, namespace, targetPodName, "test", []string{"cat", "/tmp/shared/installer/auth/kubeadmin-password"})
+	}
 	if err != nil {
 		klog.Infof("error: Job %q unable to get kubeadmin password: %v", job.Name, err)
```

Comment:
I don't know if we care enough to preserve the error from the first read attempt to include in this log message as well.